concepts/regular-expressions/about.md

```diff
@@ -0,0 +1,159 @@
+# About
+
+## Regular Expressions in JavaScript
+
+A [Regular Expression][mdn-regular-expressions] (or Regex) is a sequence of characters that we can use to target and manipulate certain elements in strings. Hence, we can:
+
+- **Search** for a text in a string
+- **Replace** substrings in a string
+- **Extract** information from a string
+
+> 💡 JavaScript's regex flavor is part of the ECMA-262 standard for the language. This means that you don't have to worry about browser-specific compatibility.
+
```

Comment:
I would recommend to add a sentence here like:
"This concept is focused on using regular expressions in JavaScript. If you want to learn about how to write regular expressions, [Eloquent JavaScript](https://eloquentjavascript.net/09_regexp.html) might be a good resource."

(The link needs to be converted to our link format we use.)

Reply:
Thanks for the recommendation! I also think it is a great resource that will complement perfectly the exercise. 

I'll add that to the Introduction.